metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/metainfo.xml

```diff
@@ -187,6 +187,14 @@
                   <timeout>600</timeout>
                 </commandScript>
               </customCommand>
+              <customCommand>
+                <name>ZEPPELIN_DASHBOARD_INSTALL</name>
```

Comment:
Seems like they use "Import" instead of "Install".  How about "Zeppelin Notebook Import"?